hil/commands/admin.py

```diff
@@ -1,16 +1,96 @@
 """Implement the hil-admin command."""
-from hil import config, model
+from hil import config, model, deferred, server, migrations, rest
 from hil.commands import db
 from hil.commands.migrate_ipmi_info import MigrateIpmiInfo
 from hil.commands.util import ensure_not_root
 from hil.flaskapp import app
+
+from time import sleep
 from flask.ext.script import Manager
 
+import sys
+import logging
+from click import IntRange
+
 manager = Manager(app)
+
 manager.add_command('db', db.command)
 manager.add_command('migrate-ipmi-info', MigrateIpmiInfo())
 
 
+@manager.command
```

Comment:
could do:
`@manager.command(name='serve-networks')`

Reply:
I tried that too, but didn't work. Apparently I have to put it in a class which inherits from `flask_script.Command` and then I can do it. Will push it soon.